### DISCUSSION TITLE
Error trying to connect: Connection closed from remote host after logging in to Firefly latest version

### DISCUSSION THREAD

**Discussion by cedricdonigarten** [iotaledger/firefly#8073] [Help & Support]
Mon, 19 Feb 2024 16:45:16 GMT
Generic: {"type":"error","payload":{"type":"client","error":"node error: error sending request for url (https://iota-node.tanglebay.com/api/core/v2/info): error trying to connect: Eine vorhandene Verbindung wurde vom Remotehost geschlossen. (os error 10054)"}}

**Comment by begonaalvarezd** [comment]
It looks like your system is blocking the connection to the node, if you are using a VPN try to turn it on & off (try on both) and if you have a firewall please check how limitting it is
